stable/anchore-engine/templates/enterprise_feeds_deployment.yaml

```diff
@@ -43,7 +43,7 @@ spec:
       {{- with .Values.anchoreEnterpriseFeeds.annotations }}
         {{ toYaml . | nindent 8 }}
       {{- end }}
-      {{- if not .Values.inject_secrets_via_env }}
+      {{- if not (or .Values.inject_secrets_via_env .Values.anchoreEnterpriseUi.existingSecret) }}
```

Comment:
It looks like there is a typo in this line. The existing secret for the feeds service comes from `.Values.anchoreEnterpriseFeeds.existingSecret`